internal/bminventory/inventory.go

```diff
@@ -259,6 +259,8 @@ func (b *bareMetalInventory) UpdateDiscoveryIgnitionInternal(ctx context.Context
 		return common.NewApiError(http.StatusBadRequest, err)
 	}
 
+	b.eventsHandler.AddEvent(ctx, params.ClusterID, nil, models.EventSeverityInfo, "Custom ignition config was applied to the cluster.", time.Now())
```

Comment:
I feel like this should happen after the update runs.

Reply:
Also maybe say "discovery ignition" here

Reply:
> I feel like this should happen after the update runs.

Yeah, I thought about it. There're many reasons why the update function may fail. Hopefully it resulted in a transaction rollback. But if not, I prefer to get an event even if the customization wasn't persisted than the other way around. WDYT?

Reply:
Actually I really only want an event if the customization was used.
I think the purpose would be to signal me to go look into the cluster/host column to see if whatever is there could break things.
But if there's an event, but nothing in the db it's just a waste of time.